### ISSUE TITLE
Load Default Profile

### ISSUE THREAD

**Issue by marcocucinato** [thegouger/MIDI2LR#19]
MIDI2LR starts always with an empty profile; A valid profile must be loaded before starting working.
It would be nice to have a default or last profile automatically loaded on startup.

**Comment by thegouger** [comment]
Done here: 3f0e479104b88c3ed8cf3cd6e0502d820c6b4da3
